charts/opentelemetry-ebpf/values.yaml

```
@@ -0,0 +1,274 @@
+# Default values for opentelemetry-ebpf.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+clusterName: ""
+
+image:
+  tag: ""
+  registry: otel
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+
+resources: {}
+
+endpoint:
+  address: ""
+  port: ""
```

We should define default here once instead of in all the templates
```suggestion
  port: 4317
```

---

Can be addressed in a follow-up PR